src/applications.rs

```diff
@@ -14,11 +17,42 @@ impl vr::IVRApplications007_Interface for Applications {
     }
     fn LaunchInternalProcess(
         &self,
-        _: *const c_char,
-        _: *const c_char,
-        _: *const c_char,
+        pch_binary_path: *const c_char,
+        pch_arguments: *const c_char,
+        pch_working_directory: *const c_char,
```

Comment:
```suggestion
        binary_path: *const c_char,
        arguments: *const c_char,
        working_directory: *const c_char,
```
hungarian notation makes me ill